src/cromshell/utilities/CromshellConfig.py

```diff
@@ -0,0 +1,91 @@
+from pathlib import Path
+import logging
+
+class CromshellConfig(object):
+    """Setup Cromshell config details"""
+
+
+
+    @classmethod
+    def user_defined_slim_metadata_parameters(cls, slim_metadata_parameters_user):
+        """Set Cromwell Slim Metadata From Command Line"""
+        if slim_metadata_parameters_user:
+            cls.slim_metadata_parameters = slim_metadata_parameters_user + "&includeKey=subWorkflowMetadata&includeKey=subWorkflowId"
+
+    @classmethod
+    def override_cromwell_config_server(cls, server_user=None, workflow_id=None):
+        """Set Cromwell Server From Command Line or Environment or Submission file"""
```

Comment:
A bit more explanation is justified, I think, that
 * why two parameters are specified, 
 * what are the intended usage scenarios, 
 * what are the expected behaviors of the method.

Reply:
Added a description, may need a second look

Reply:
We can talk about exactly how to word this.
Clearly laying out the expected usage scenarios helps users.
I think the doc now has that, just needs a bit more structuring.